planner/planner.py

```diff
@@ -128,7 +129,8 @@ def planner_pipelines():
         'update_time': update_time,
         'dependencies': dependencies,
         'pipeline': steps(*final_steps),
-        'hooks': [FLOWMANAGER_HOOK_URL]
+        'hooks': [FLOWMANAGER_HOOK_URL],
+        'title': 'Creating Package'
```

Comment:
Not sure why putting this in the pipeline details is the way to go.
Waiting to see how it's used in the specstore.